samples/peripheral/radio_test/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8.2)
+
```

Comment:
This is only a suggestion. You can add here a list of supported boards.
set(NRF_SUPPORTED_BOARDS
  nrf5340_dk_nrf5340_cpunet
  nrf52_pca10040
  nrf52840_pca10056
)

This causes that the sample can be build only for listed platform.